core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -406,10 +407,17 @@ public static Duration getDuration(PropertyKey key) {
   }
 
   /**
-   * @return a view of the internal {@link Properties} of as an immutable map
+   * @return a view of the internal {@link Properties} and default properties of as an immutable map
```

Comment:
The previous one looks strange, I changed to "a view of the properties(include default properties) of as an immutable map".  Would it sound better?

Reply:
I think that's better, how about "a copy of the configuration (including default properties) as an immutable map"? I switched "view" to "copy" since "view" suggests that we don't make a copy.

Reply:
oh actually Collection.unmodifiableMap does return a view, we should stick with calling it a view